src/api/messages.ts

```diff
@@ -21,3 +22,28 @@ export const postMessage = async (formData: {
     };
   }
 };
+
+export const getMessages = async (isApproved?: boolean | null) => {
+  const data =
+    (await apiGet<Message[]>(
+      `messages/email/received-owner-requests?approvedFilter=${isApproved}`,
+    )) || [];
```

Comment:
I believe the API does return empty array when nothing is found, doesn't it?

Reply:
Might be, but never trust it, though.

Reply:
Legit